packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -676,12 +683,8 @@ export const DatePickerMixin = (subclass) =>
 
     /** @protected */
     _openedChanged(opened) {
-      if (opened && !this._overlayInitialized) {
-        this._initOverlay();
-      }
-      if (this._overlayInitialized) {
-        this.$.overlay.opened = opened;
-      }
+      this.$.overlay.opened = opened;
```

Comment:
This can be removed since we have `opened` in the template, I guess.

Reply:
Thanks, done.